dataset_builders/pie/abstrct/README.md

```diff
@@ -92,22 +105,236 @@ Morio et al. ([2022](https://aclanthology.org/2022.tacl-1.37.pdf); p. 642, Table
 
 (Mayer et al. 2020, p.2110)
 
-#### Examples
+#### Example
 
-![Examples](img/abstr-sam.png)
+![abstr-sam.png](img%2Fabstr-sam.png)
 
-### Document Converters
+### Collected Statistics after Document Conversion
 
-The dataset provides document converters for the following target document types:
+In this section, we collect further statistics of the dataset after the conversion to `TextDocumentWithLabeledSpansAndBinaryRelations`.
+In the commands, we used the following dataset config: `configs/dataset/abstrct_base.yaml`:
 
-- `pytorch_ie.documents.TextDocumentWithLabeledSpansAndBinaryRelations`
-  - `labeled_spans`: `LabeledSpan` annotations, converted from `BratDocumentWithMergedSpans`'s `spans`
-    - labels: `MajorClaim`, `Claim`, `Premise`
-  - `binary_relations`: `BinaryRelation` annotations, converted from `BratDocumentWithMergedSpans`'s `relations`
-    - labels:  `Support`, `Partial-Attack`, `Attack`
+```commandline
+_target_: src.utils.execute_pipeline
+input:
+  _target_: pie_datasets.DatasetDict.load_dataset
+  path: pie/abstrct
+  revision: 277dc703fd78614635e86fe57c636b54931538b2
+```
 
-See [here](https://github.com/ChristophAlt/pytorch-ie/blob/main/src/pytorch_ie/documents.py) for the document type
-definitions.
+The script `evaluate_documents.py` comes from [PyTorch-IE-Hydra-Template](https://github.com/ArneBinder/pytorch-ie-hydra-template-1).
+
+For the tokenization, we use `bert-base-uncased` from `transformer.AutoTokenizer` (see [AutoTokenizer](https://huggingface.co/docs/transformers/v4.37.1/en/model_doc/auto#transformers.AutoTokenizer), and [bert-based-uncased](https://huggingface.co/bert-base-uncased))
+to tokenize `text` in `TextDocumentWithLabeledSpansAndBinaryRelations` (see [document type](https://github.com/ChristophAlt/pytorch-ie/blob/main/src/pytorch_ie/documents.py)).
+
+#### Relation argument (outer) token distance per label
+
+The distance is measured from the first token of the first argumentative unit to the last token of the last unit, a.k.a. outer distance.
+
+We collect the following statistics: number of documents in the split (*no. doc*), no. of relations (*len*), mean of token distance (*mean*), standard deviation of the distance (*std*), minimum outer distance (*min*), and maximum outer distance (*max*).
+We also present histograms in the collasible, showing the distribution of these relation distances (x-axis; and unit-counts in y-axis), accordingly.
+
+*Note that*: to collect the relation argument distance by tokens, the [respective branch](https://github.com/ArneBinder/pytorch-ie-hydra-template-1/pull/135) must be checked out instead of the `main` branch.
```

Comment:
```suggestion
**Important**: To reproduce the statistics and figures for this metric by yourself, the [`relation_argument_distance_collector` branch](https://github.com/ArneBinder/pytorch-ie-hydra-template-1/pull/135) of `pytorch-ie-hydra-template` needs to be checked out instead of the `main` branch!

Reply:
edited as suggested in [d4167ce](https://github.com/ArneBinder/pie-datasets/pull/100/commits/d4167ce64e3e10193f28756078a6b271a862e532)

Reply:
I moved the implementation of the relation argument distance collector to `pie-modules`. Can you look at https://github.com/ArneBinder/pie-modules/pull/71 and finish it so that we do not need that comment at all?